src/cbapi/psc/rest_api.py

```diff
@@ -27,3 +37,101 @@ def live_response(self):
 
     def _request_lr_session(self, sensor_id):
         return self.live_response.request_session(sensor_id)
+
+    #---- Device API
+    
+    def get_device(self, device_id):
+        """
+        Locate a device with the specified device ID.
+        
+        :param int device_id: The ID of the device to look up.
+        :return: The new device object.
+        """
+        rc = Device(self, device_id)
+        rc.refresh()
+        return rc
+    
+    def _raw_device_action(self, request):
```

Comment:
Why did we put all of device action at the PSCBaseAPI instead of inside the Device model?

Reply:
They're in _both_ places.  The PSCBaseAPI versions can work on multiple devices by ID.

Reply:
The query has to hook into the same functionality as well.  Since the API object is kind of the "basis" for all the functionality, it's the most logical place to put the "shared" code.

Reply:
I am a little concerned that the BaseAPI objects will be the catch all for everything in the future. Would it make sense to have a utils directory with common code like this? 

Do we want the experience to be `cb.device_bypass(id)` or `cb.select(Device, id).bypass`?
Here is the example I am referring to for getting a device by id?

Reply:
I believe you, but then, how do we support the use case of changing the status of multiple devices by id at once?  Remember, the syntax is not cb.device_bypass(id), it's cb.device_bypass([ id ]). This allows us to also do cb.device_bypass([ id1, id2, id3 ]).

Reply:
After discussion, neither Alex nor I can figure out how to do this any better.